core/src/impl/Kokkos_HostSharedPtr.hpp

```diff
@@ -47,6 +47,7 @@
 
 #include <Kokkos_Macros.hpp>
 #include <Kokkos_Atomic.hpp>
+#include <impl/Kokkos_Error.hpp>
```

Comment:
Why do we need this include?

Reply:
https://github.com/kokkos/kokkos/blob/9fee92f46596334850edcf4a099f1a76eb084c53/core/src/impl/Kokkos_HostSharedPtr.hpp#L130